.github/workflows/mac_binary.yml

```
@@ -4,7 +4,8 @@ on:
     branches:
       - master
     tags:
-      - '[0-9]+.[0-9]+.[0-9]+.*'
+      - '[0-9]+.[0-9]+.[0-9]+*'
+    workflow_dispatch:
```

```suggestion
  workflow_dispatch:
```

I believe, this needs to be `on.workflow_dispatch` not `on.push.workflow_dispatch`.

---

Ideally this is put in between `on` and `push` so it is easier to keep track of identation.